github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Todos
+- [ ] Tests
+- [ ] Documentation
+- [ ] CHANGELOG
+
+## Deploy Notes
+Notes regarding deployment the contained body of work. These should note any
```

Comment:
`these should note` -> `These should highlight` (so we don't repeat 'note' too often)